src/commands/clone-backend.ts

```diff
@@ -0,0 +1,62 @@
+import {BaseCommand} from '../lib'
+import {flags} from '@oclif/command'
+import {cli} from 'cli-ux'
+
+const RESTORE_MUTATION  = `
+mutation($uid: String!) {
+  restore(uid: $uid) {
+    response { code message restoreId }
+  }
+}`
+
+export default class CloneBackend extends BaseCommand {
+    static description = 'Clone into a backend by source backend ID'
+
+    static hidden = true;
+
+    static examples = [
+      '$ slash-graphql clone-backend -e https://clone.cloud.dgraph.io/graphql -t <apiToken> --source <source backend id or url>',
+    ]
+
+    static flags = {
+      ...BaseCommand.commonFlags,
+      ...BaseCommand.endpointFlags,
+      source: flags.string({char: 's', description: 'Source backend ID or url to get the data to be cloned', required: true}),
+      confirm: flags.boolean({char: 'y', description: 'Skip Confirmation', default: false}),
+    }
+
+    confirm() {
+      this.log('This will destroy your backend data and overwrite with the source backend data.')
+      return cli.confirm('Are you sure you want to proceed?')
+    }
+
+    async run() {
+      const opts = this.parse(CloneBackend)
+      const backend = await this.backendFromOpts(opts)
+      let sourceID = opts.flags.source
+
+      if (!sourceID.match(/0x[0-9a-f]+/)) {
+        sourceID = backend.getUid()
+      }
+
+      if (!(opts.flags.confirm || await this.confirm())) {
+        this.log('Aborting')
+        return
+      }
+
+      const {errors, data} = await backend.slashAdminQuery<{updateGQLSchema: {gqlSchema: {schema: string}}}>(RESTORE_MUTATION, {
+        uid: sourceID,
+      })
+
+      if (errors) {
+        for (const {message} of errors) {
+          this.error(message)
+        }
+        return
+      }
+
+      const restoreId = data.restore.response.restoreId
+
+      this.log('Cloning in progress. Use the "clone-backend-status" command to fetch the status using the key', restoreId)
```

Comment:
I would say "Restore in progress. Please run `slash-graphql restore-status` to fetch the current status, using the following key"

If flags.quiet, then put the whole message. Else just log restoreId.